src/main/java/seedu/address/logic/commands/addcommand/AddParticipantCommand.java

```diff
@@ -2,36 +2,49 @@
 
 import static java.util.Objects.requireNonNull;
 
+import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
+import seedu.address.model.entity.Name;
 import seedu.address.model.entity.Participant;
 
 /**
  * Adds a {@link Participant} to Alfred.
  */
 public class AddParticipantCommand extends AddCommand {
 
-    /* Possible Fields */
     public static final String COMMAND_WORD = "addParticipant";
     private static final String MESSAGE_SUCCESS = "New participant added: %s";
     private static final String MESSAGE_DUPLICATE_PARTICIPANT = "This participant already exists in this Hackathon";
 
     private Participant participant;
+    private Name teamName;
 
     public AddParticipantCommand(Participant participant) {
         requireNonNull(participant);
         this.participant = participant;
     }
 
+    public AddParticipantCommand(Participant participant, Name teamName) {
```

Comment:
Does this mean that the "add participant to team" command is basically integrated into this AddParticipantCommand?

Reply:
Yea I was thinking if the user inputs something like "add participant t/teamName" then we'll add the participant to specified team. And since we're adding a Participant to somewhere, I figured we don't have to make another class for it. wdyt

Reply:
wait wait but if you're saying inputting "add participant t/teamName" how is the user specifying which participant he's adding to the specific team?

Reply:
Oh yea gotta specify participant's name too. All ears for any other ideas tho

Reply:
Actually may be quite hard though, since adding participant includes other details like email and phone number.
Maybe everytime the user adds participant, there can be a prompt to tell them to use (addTeam participantName/participantId teamName)? haha idk

Reply:
Hey i think you would need to change this to search for existing participant and add them to the team respectively. 😄